### DISCUSSION TITLE
permission denied

### DISCUSSION THREAD

**Discussion by gramashawana** [community/community#37239] [Other Feature Feedback, Questions, & Ideas]
fatal: unable to access 'https://github.com/gramashawana/printf.git/': The requested URL returned error: 403

**Comment by msuliq** [comment]
Possible causes:  you typed an incorrect password. Make sure you are using the correct GitHub password for the account.

If you are sure you are using the correct GitHub password, try using SSH. For information about how to do this, please visit https://help.github.com/articles/changing-a-remote-s-url.

**Reply by airtower-luna** [reply]
You can't use your account password for Git operations, you have to use a PAT with appropriate scopes.

**Reply by gramashawana** [reply]
thanks

**Comment by airtower-luna** [comment]
403 means that access was forbidden, so you didn't provide the right authorization (a token for Git over HTTPS). Without more details it's hard to say exactly why, but: Make sure you're using a token with the right scopes for what you're trying to do (e.g. reading the repository, writing the repository, editing workflow files if you use Actions). If you have trouble with that, please quote the full command and error message so people can give better advice.

**Reply by gramashawana** [reply]
thanks
